changelogs/CHANGELOG-v0.5.2.md

```diff
@@ -0,0 +1,55 @@
+The v0.5.2 release is a patch release that updates bpfman to aya version 0.13.0
+and aya-obj version 0.2.0.  This update allows bpfman to make a proper bpfman
+release that doesn't have Aya pined to a git SHA.
+
+Other Notable Changes:
+* Add support for ppc and S390
+* Allow Cosign to be disabled and enabled at runtime.
+* Added an OpenSSF Scorecard action to scan bpfman for security best practices
+  on a regular basis and started making recommended improvements.
+
+## What's Changed
+* Add bpfman OpenShift container to build bpfman by @msherif1234 in
+  https://github.com/bpfman/bpfman/pull/1231
+* bpfman/lib: add function documentation by @frobware in
+  https://github.com/bpfman/bpfman/pull/1186
+* Move Fedora back to release 40 in Containerfile.bpfman.local by @anfredette in
+  https://github.com/bpfman/bpfman/pull/1235
+* bpfman: Make whether to use cosign configurable by @anfredette in
+  https://github.com/bpfman/bpfman/pull/1240
+* Fix BUILDPLATFORM redefinition issue in bpfman Containerfile by @msherif1234
+  in https://github.com/bpfman/bpfman/pull/1244
+* add build-source to fix release violations by @msherif1234 in
```

Comment:
lets drop all konflux related works from the changeset so this and the following one

Reply:
Will do.  I deleted all of the ones done by the konflux bot, but not the konflux related ones by you.  But it's probably good to delete those too.